src/shillelagh/filters.py

```diff
@@ -425,3 +426,36 @@ def __repr__(self) -> str:
             operator = "<=" if self.include_end else "<"
             comparisons.append(f"{operator}{self.end}")
         return ",".join(comparisons)
+
+
+class In(Filter):
+    """
+    Substring searches.
+    """
+
+    operators: Set[Operator] = {
+        Operator.IN,
+    }
+
+    def __init__(self, value: Any):
+        self.value = value
+
+    @classmethod
+    def build(cls, operations: Set[Tuple[Operator, Any]]) -> Filter:
+        # we only accept a single value
+        values = {value for operator, value in operations}
+        if len(values) != 1:
+            return Impossible()
+
+        # we only accept tuples
+        value = values.pop()
```

Comment:
why pop here?

Reply:
Copied the concept from all the other filters. I'll remove the check for 'tuple' below and then normalize it with all the others.

Reply:
@betodealmeida can comment on the pop and the rest of this code.